config/base.keymap

```diff
@@ -73,7 +64,7 @@ ZMK_COMBO(c_end, &kp END, RM2 RM3, ALL, COMBO_JOINTLY_MS)
 ZMK_COMBO(c_pgdn, &kp PG_DN, RB1 RB2, ALL, COMBO_JOINTLY_MS)
 ZMK_COMBO(c_supup, &kp LG(UP), LT3 LT2 LT1, ALL, COMBO_JOINTLY_MS)
 ZMK_COMBO(c_supleft, &kp LG(LEFT), LM3 LM2 LM1, ALL, COMBO_JOINTLY_MS)
-ZMK_COMBO(c_supright, &kp LG(RIGHT), LM2 LM1 LMB0, ALL, COMBO_JOINTLY_MS)
+ZMK_COMBO(c_supright, &kp LG(RIGHT), LM2 LM1 LM0, ALL, COMBO_JOINTLY_MS) // Was LMB0. Assumed LM0
```

Comment:
This one I noticed when trying a KMK port and sighed deeply. Ope. Indeed, LM0 is intended here. Ty.